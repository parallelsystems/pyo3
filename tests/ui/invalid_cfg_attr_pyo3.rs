#![cfg(feature = "macros")]

use pyo3::prelude::*;

#[cfg_attr(feature = "pyo3", pyclass)]
struct Foo {
    #[cfg_attr(feature = "pyo3", pyo3(get), fail)]
    x: i32,
}

fn main() {
    Foo { x: 1 };
}
