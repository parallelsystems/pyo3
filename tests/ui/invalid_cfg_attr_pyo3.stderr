error: cannot find attribute `fail` in this scope
 --> tests/ui/invalid_cfg_attr_pyo3.rs:7:45
  |
7 |     #[cfg_attr(feature = "pyo3", pyo3(get), fail)]
  |                                             ^^^^
